tests/unit/test_api.py

```diff
@@ -123,3 +124,21 @@ def test_sync_release_version_tag_processing(
         api_mock.sync_release(
             version=version or get_version_return, tag=tag, dist_git_branch="_"
         )
+
+
+@pytest.mark.parametrize(
+    "path, downstream_package_name, expectation",
+    [
+        pytest.param("/systemd", "systemd", "systemd", id="both_set"),
+        pytest.param(None, "systemd", "systemd", id="both_set"),
+        pytest.param("/systemd", None, "systemd", id="both_set"),
+        pytest.param(None, None, None, id="both_set"),
```

Comment:
I don't think the `id`s are correct.

Reply:
thanks, I added as a last thing before even trying it out and I believe tests are failing on this as well; will fix